examples/validation_tests.ipynb

```diff
@@ -0,0 +1,558 @@
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "fig, axs = plt.subplots (2, sharex=True, gridspec_kw={'hspace': 0},figsize=(8,8))\n",
+    "\n",
+    "axs[0].plot (r, np.abs (ccl_Sigma_ein / nc_Sigma_ein - 1.0), \n",
+    "             label = \"CCL - EIN\", linestyle='-', color='darkgreen')\n",
+    "axs[0].plot (r, np.abs (ccl_Sigma_her / nc_Sigma_ein - 1.0),\n",
+    "             label = \"CCL - HER\", linestyle='--', color='darkgreen')\n",
+    "axs[0].plot (r, np.abs (ct_Sigma_ein / nc_Sigma_ein - 1.0), \n",
+    "             label = \"CT - EIN\", linestyle='-', color='purple')\n",
+    "# axs[0].plot (r, np.abs (col_Sigma_ein / nc_Sigma_ein - 1.0), '-', \n",
+    "#              label = \"EIN - COL\", linestyle='--', color='darkgrey')\n",
+    "axs[1].plot (r, np.abs (ccl_DeltaSigma_ein / nc_DeltaSigma_ein - 1.0), \n",
+    "             label = \"CCL - EIN\", linestyle='-', color='darkgreen')\n",
+    "axs[1].plot (r, np.abs (ccl_DeltaSigma_her / nc_DeltaSigma_ein - 1.0),\n",
+    "             label = \"CCL - HER\", linestyle='--', color='darkgreen')\n",
+    "axs[1].plot (r, np.abs (ct_DeltaSigma_ein / nc_DeltaSigma_ein - 1.0),\n",
+    "             label = \"CT - EIN\", linestyle='-', color='purple')\n",
+    "# axs[1].plot (r, np.abs (col_DeltaSigma_ein / nc_DeltaSigma_ein - 1.0), '-', \n",
+    "#              label = \"EIN - COL\", linestyle='--', color='darkgrey')\n",
+    "\n",
+    "axs[0].set_xscale ('log')\n",
+    "axs[0].set_yscale ('log')\n",
+    "axs[1].set_xlabel('R [Mpc]', fontsize=14)\n",
+    "axs[0].set_ylabel(r'$|\\Sigma_{\\mathrm{i}} / \\Sigma_{\\mathrm{NC}} - 1|$', fontsize=16)\n",
+    "axs[1].set_xscale ('log')\n",
+    "axs[1].set_yscale ('log')\n",
+    "axs[1].set_ylabel(r'$|\\Delta\\Sigma_{\\mathrm{i}} / \\Delta\\Sigma_{\\mathrm{NC}} - 1|$', fontsize=16)\n",
+    "\n",
+    "axs[1].tick_params(labelsize=14)\n",
+    "axs[0].tick_params(labelsize=1)\n",
+    "axs[0].legend(fontsize=12)\n",
+    "fig.tight_layout()\n",
+    "fig.savefig('Einasto_precision.pdf')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Compare all to Colossus"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
+    "fig, axs = plt.subplots (2, sharex=True, gridspec_kw={'hspace': 0}, figsize=(10,8))\n",
+    "\n",
+    "axs[0].set_xscale ('log')\n",
+    "axs[0].set_yscale ('log')\n",
+    "axs[1].set_yscale ('log')\n",
+    "axs[1].set_xlabel('R (Mpc)')\n",
+    "axs[0].set_ylabel(r'$\\Sigma$ [M$_\\odot$  Mpc$^{-2}$]')\n",
+    "axs[1].set_ylabel(r'$\\Sigma_{\\mathrm{i}} (R) / \\Sigma_{\\mathrm{col}} (R) - 1$')\n",
+    "axs[0].plot (r, col_Sigma_nfw, '-', label = 'NFW - Col', color = 'lightcoral')\n",
+    "axs[0].plot (r, col_Sigma_ein, '-', label = 'EIN - Col', color = 'cadetblue')\n",
+    "axs[0].plot (r, col_Sigma_her, '-', label = 'HER - Col', color = 'tan')\n",
+    "\n",
```

Comment:
The following 11 lines are giving me a lot of warnings about linestyle is passed as kwarg, so the positional argument '-' will be ignored.

Reply:
DOne